File: 0070-climbing-stairs/0070-climbing-stairs.py
class Solution(object):
    def climbStairs(self, n):
        """
        :type n: int
        :rtype: int
        """
        # Fibonacci Logic
        a = 0
        b = 1
        for _ in range(n):
            a, b = b, a+b
        return b

File: 0070-climbing-stairs/0070-climbing-stairs.rs
impl Solution {
    pub fn climb_stairs(n: i32) -> i32 {
        let mut a = 1;
        let mut b = 1;
        for _ in 2..n+1{
            (a, b) = (b, a+b);
        }
        return b;
    }
}